textproc/jet/Portfile

```diff
@@ -0,0 +1,67 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           github 1.0
+
+github.setup        borkdude jet 0.6.26 v
+github.tarball_from releases
+revision            0
+
+maintainers         {@catap korins.ky:kirill} openmaintainer
+license             EPL-1
+categories          textproc devel
+
+description         CLI to transform between JSON, EDN, YAML and Transit using Clojure
+long_description    {*}${description}
+
+distfiles           ${name}-${version}-standalone.jar
+
+checksums           rmd160  7730a89dbb9117d066a633aad51cfbad6a6d5a43 \
+                    sha256  ce5488219d1857abacb20032973f101a62818bc3302083339a5a087f86c7bc75 \
+                    size    13531702
+
+extract {}
+
+variant jdk17 conflicts jdk11 description {Build using JDK 17} {
+    depends_build-append port:openjdk17-graalvm
+}
+
+variant jdk11 conflicts jdk17 description {Build using the older JDK 11} {
+    depends_build-append port:openjdk11-graalvm-native-image
+}
+
+default_variants    +jdk17
+use_configure       no
+
+# Leaving this stub since we cannot declare global variables in the scope of pre-fetch
+set graalvm_home /Library/Java/JavaVirtualMachines
```

Comment:
Why does it need to be a global variable? It's only used in the build block. It could be declared in there.

Reply:
no reason, thanks for catching it.